packages/launch/src/focused-launch/index.tsx

```diff
@@ -2,26 +2,32 @@
  * External dependencies
  */
 import React from 'react';
-import { MemoryRouter as Router, Switch, Route } from 'react-router-dom';
+import { MemoryRouter as Router, Switch, Route, Redirect } from 'react-router-dom';
 
 /**
  * Internal dependencies
  */
+import { useSite } from '../hooks';
 import { Route as FocusedLaunchRoute } from './route';
 import Summary from './summary';
 import DomainDetails from './domain-details';
 import PlanDetails from './plan-details';
 import Success from './success';
-import { useOnLaunch } from '../hooks';
 
 import './style.scss';
 
 const FocusedLaunch: React.FunctionComponent = () => {
-	// handle redirects to checkout / my home after launch
-	useOnLaunch();
+	const { isLaunched, isSiteLaunching } = useSite();
+
+	React.useEffect( () => {
+		if ( isLaunched || isSiteLaunching ) {
+			document.body.classList.add( 'is-focused-launch-complete' );
+		}
+	}, [ isLaunched, isSiteLaunching ] );
 
 	return (
 		<Router initialEntries={ [ FocusedLaunchRoute.Summary ] }>
+			{ ( isLaunched || isSiteLaunching ) && <Redirect to={ FocusedLaunchRoute.Success } /> }
```

Comment:
It looks like the "Success" view now is more of a "Launch complete" view. We should at least add a comment explaining that
- the `Success` view is shown also while the site is still launching
- the `Success` view is technically going to be the selected view in the modal even while the user goes through the checkout flow (which is rendered on top of this view)

In the long term, we should probably rename the `Success` view/route to reflect its new usage

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/47601/commits/7f18ef996c0a7d6476887aac7302b9d9036019fa